apps/nextjs/src/app/admin/aila/[chatId]/view.tsx

```diff
@@ -41,14 +45,16 @@ function ModerationListItem({ moderation }: { readonly moderation: Moderation })
             {moderation.justification}
           </blockquote>
           <div className="mt-2 space-x-2">
-            {moderation.categories.map((category, index) => (
-              <span
-                key={index}
-                className="inline-block rounded-md bg-zinc-300 px-8 py-4 text-xs font-semibold text-zinc-800"
-              >
-                {String(category)}
-              </span>
-            ))}
+            {Array.from(new Set(moderation.categories))
+              .map((c) => String(c))
+              .map((category) => (
+                <span
+                  key={category}
```

Comment:
Is this always unique?

Reply:
Yes, we have a set of categories that the moderation can flag